src/_locales/zh/messages.json

```diff
@@ -192,6 +192,8 @@
     "set_as_default": "設為默認",
     "set_up": "開始",
     "settings": "設定",
+    "settings_advance_security": "高級安全模式",
+    "settings_advance_security_desc": "將内容展示在影子根中，以免於社交網路之窺探。",
```

Comment:
No `settings_vim_compat` here?

Reply:
following the suggestion from #478 @neruthes

Reply:
I mean `settings_vim_compat` exists in the English version but absents here.